Helpers/OpenT2T-Wink-Helper/js/index.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+var https = require('https');
+var q = require('q');
+
+//this is our base, we refactor these options in each method
+
+
+var  accessToken,ApiPath;  //we will need this when creating the headers
```

Comment:
JavaScript naming convention is to only capitalize constructors/class names. So make this lowercase.